input_testdata/mitx.01/about/overview.html

```diff
@@ -0,0 +1 @@
+hello overview
```

Comment:
Yeah, I copied a file from an xbundle in a LORE PR and Giovanni asked for it to be fixed. I was just wondering why it's flagged if it doesn't affect functionality.